.github/workflows/auto-author-assign.yml

```diff
@@ -5,14 +5,16 @@ on:
     types: [opened, reopened]
 
 env:
-  IS_FORK: ${{ github.event.pull_request.head.repo.full_name != github.repository }}
+  JOB_ALLOWED_OWNER:
```

Comment:
имя переменной стало сильно непонятнее. Возможно стоит ввести две переменные: в старой IS_FORK проверять на форк и еще одну на автора

Reply:
I tried to do that:

```
env: 
  IS_FORK: ${{ ... }}
  IS_DEPENDABOT: ${{ ... }}
  JOB_ALLOWED_OWNER: ${{ !Ienv.S_FORK && !env.IS_DEPENDABOT }}
```

but in this case, a syntax error occurs because the env is not available before

Reply:
вообще убери JOB_ALLOWED_OWNER

Reply:
I originally did this, but decided to follow the comments)

![image](https://user-images.githubusercontent.com/12021443/150299408-4b784348-284e-41b3-8466-d1e349d52e85.png)

Reply:
@IKatsuba updated

Reply:
мне показалось излишне проверять везде на обе переменные, если они за одно и тоже отвечают и одна без другой все равно не бывает) особенно в lint.yml куча повторений